docs/ga-roadmap.md

```diff
@@ -1 +1,54 @@
 # Collector v1 Roadmap
+
+This document contains the roadmap for the Collector as of 4-Apr-2024. The main goal of this roadmap is to provide clarity on the areas of focus in order to release a v1 of the Collector.
+
+## Proposal
+
+The proposed approach to delivering a stable release of the OpenTelemetry Collector is to produce a distribution of the Collector that contains a minimum set of components which have been stabilized. By doing so, the project contributors will ensure dependencies of those components have also been released under a stable version.
+
+The proposed distribution is set to include the following components only:
+
+- OTLP receiver
+- OTLP exporter
+- OTLP HTTP exporter
+
+These modules depend on a list of other modules, the full list is available in issue [#9375](https://github.com/open-telemetry/opentelemetry-collector/issues/9375).
+
+## Out of scope
+
+Explicitly, the following are not in the scope of v1 for the purposes of this document:
+
+* stabilization of all component developer APIs or other parts of the Collector. Component developers are NOT the audience for this work
```

Comment:
Let's define what we mean by `component developer`. Are the core modules are required by component developers, and those are what we are stabilizing, so this exclusion isn't clear to me.

Reply:
You're right, i do think some of the APIs used by component developers are being stabilized as part of the work being done. Removing this line